bounceview/src/main/java/com/github/hariprasanths/bounceview/CustomPopupDialog.java

```diff
@@ -0,0 +1,90 @@
+package com.github.hariprasanths.bounceview;
+
+import ohos.agp.colors.RgbColor;
+import ohos.agp.components.Component;
+import ohos.agp.components.DirectionalLayout;
+import ohos.agp.components.LayoutScatter;
+import ohos.agp.components.Text;
+import ohos.agp.components.element.Element;
+import ohos.agp.components.element.ShapeElement;
+import ohos.agp.utils.Color;
+import ohos.agp.window.dialog.PopupDialog;
+import ohos.app.Context;
+
+/**
+ * Custom Popup Dialog.
+ */
+public class CustomPopupDialog extends PopupDialog {
+    Component mComponent = null;
+    Component mTempComponent = null;
```

Comment:
rename this variable to ### mBaseComponent

Reply:
done